### ISSUE TITLE
Maryan's Research

### ISSUE THREAD

**Issue by loretopaulinojr** [loretopaulinojr/CCOMP#10]
## Discuss Moran, Kujawinski, et al. 2022 – Microbial metabolites in the marine carbon cycle

As you read, keep a running list of unfamiliar vocabulary. Look up the terms and keep notes. 

Work carefully through the figures and be prepared to discuss them.

Look for the knowledge gaps the authors highlight and be prepared to discuss them. Are there any other gaps that you see that are not highlighted in the review?

**Comment by loretopaulinojr** [comment]
# Redox Imbalance and Extracellular Release

The passage suggests that, at certain points in the diel cycle (daily cycle) and specific regions in the photic zone (upper layer of water where sunlight penetrates), extracellular release from phytoplankton may occur in response to a redox imbalance.

A redox imbalance could be caused by factors such as changes in light conditions (light shock), which may affect the balance of electron transfer reactions within the cells.

## Factors Driving Extracellular Release

The text mentions nutrient stress as a potential driver for organic carbon excretion. When cells increase their ATP/ADP ratio (indicating a higher intracellular energy state) in response to nutrient stress, they may release organic carbon as a by-product to mitigate the free-energy costs of transporting low concentration nutrients.

## Chemical Composition of Exometabolites

Regardless of the release mechanism, the passage notes that the chemical composition of phytoplankton exometabolites (compounds released into the surrounding environment) is expected to differ from the composition of endometabolites (compounds present within the cell).

Different mechanisms and drivers of release, including redox imbalance, may result in the release of metabolites with distinctive chemical structures.

## Role in Carbon Transfer

The tight coordination in daily activity patterns between phytoplankton and heterotrophic bacteria in response to the diel irradiance cycle establishes a major role for extracellular products of photosynthetic activity in carbon transfer in the surface ocean.

In other words, the compounds released by phytoplankton play a significant role in the transfer of carbon within the surface ocean ecosystem.
